examples/pxScene2d/src/rcvrcore/initGL.js

```diff
@@ -0,0 +1,420 @@
+/*
+
+pxCore Copyright 2005-2018 John Robinson
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+// TODO... 
+// * Xuse passed in url/file
+// * Xfix xsetInterval and xclearInterval
+// * Xbug when having a working gl scene and load non-existent gl scene
+// * load nonexistent gl scene blue context??
+// * Xmultiple loadings of gl scenes
+// * requestAnimationTimer
+// * fix deletionn of gl context (swith to MIke's new api)
+// * multiple opengl scenes at same time
+// * key events, onSize etc... 
+// * view events... event delegation?? to scene
+// * why do I need to pass in beginDrawing and endDraving
+// NOTE TO self... declarations with no "var" are in global namespace... danger
+// are setInterval and clearInterval polluting global namespace... ?
+// /Users/johnrobinson/code/pxgl/examples/pxScene2d/external/spark-webgl/examples/2-triangle/triangle.js
+
+var _timers = require('timers')
+var fs = require('fs')
+var path = require('path')
+var vm = require('vm')
+var _module = require('module')
+var _ws = require('ws')
+var _http = require('http')
+var _https = require('https')
+var urlmain = require("url")
+var console = require('console_wrap')
+var process = require('process')
+var ESMLoader = require('rcvrcore/ESMLoader')
+var _sparkkeys = require('rcvrcore/tools/keys.js')
+var reqOrig = require
+var {promisify} = require('util')
+var Buffer = require('buffer').Buffer
+var cachedSource = {}
+
+var sandboxKeys = ["vm", "process", "setTimeout", "console", "clearTimeout", "setInterval", "clearInterval", "setImmediate", "clearImmediate", "sparkview", "sparkscene", "sparkgles2", "beginDrawing", "endDrawing", "sparkwebgl", "sparkkeys", "sparkQueryParams", "require", "localStorage"]
```

Comment:
https://github.com/pxscene/pxCore/pull/2056 is in both _webgl and master but your new initGL.js has it missing